test/test_plugin_compatibility_gate.py

```
@@ -208,6 +217,178 @@ def test_untrustworthy_core_does_not_block_installs(self, store, monkeypatch):
         assert (path / "manifest.json").exists()
 
 
+# --------------------------------------------------------------------------
+# The git-pull update path — the one route that does not re-download
+# --------------------------------------------------------------------------
+
+_HAS_GIT = subprocess.run(
+    ['git', '--version'], capture_output=True).returncode == 0
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)